tests/auth.test.ts

```diff
@@ -1,24 +1,27 @@
 import { expect, test, type Page } from "@playwright/test";
 
-test.describe("Auth", () => {
+test.describe.serial("Auth", () => {
   let page: Page;
   test.beforeAll(async ({ browser }) => {
     page = await browser.newPage();
     await page.goto("/auth-test");
   });
+  test.afterAll(async () => {
+    await page.close();
+  });
 
   test("Renders UI conditionally based on auth state", async () => {
-    expect(page.getByText("Signed In")).not.toBeVisible();
```

Comment:
Here were awaits missing, we recommend no-floating-promises for that: https://playwright.dev/docs/best-practices#lint-your-tests